src/features/EditMovieModal/EditMovieModal.jsx

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-expressions */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import clsx from 'clsx'
+import React from 'react'
+import { useFormik } from 'formik'
+import * as Yup from 'yup'
+import { useEditMovieMutation } from '../../services/api'
+import styled from './styles.module.scss'
+
+function EditMovieModal({ handleClose, item }) {
+  const [editMovie] = useEditMovieMutation()
+
+  const formik = useFormik({
+    initialValues: {
+      title: item.title,
+      poster_path: item.poster_path,
+      genres: item.genres[0],
+      release_date: item.release_date,
+      vote_average: item.vote_average,
+      runtime: item.runtime,
+      overview: item.overview,
+    },
+    validationSchema: Yup.object({
+      vote_average: Yup.number().required('Rating must be a number!'),
+      runtime: Yup.number().required('Runtime must be a number!'),
+    }),
+    onChange: (e) => formik.setFieldValue(formik.values.genres, e.target.value),
+    onSubmit: () => {
+      editMovie({
+        id: item.id,
+        title: formik.values.title,
+        poster_path: formik.values.poster_path,
+        genres: [formik.values.genres],
+        release_date: formik.values.release_date,
+        vote_average: 1 * formik.values.vote_average,
```

Comment:
same here with long paths

if you need to transform string to number we usually use `++formik.values.vote_average`